src/actions/walletConnectActions.js

```diff
@@ -378,6 +379,10 @@ export const subscribeToWalletConnectConnectorEventsAction = (connector: WalletC
       }
 
       if (method === ETH_SIGN_TYPED_DATA || method === ETH_SIGN_TYPED_DATA_V4) {
+        if (isEmpty(params)) {
+          dispatch(setWalletConnectErrorAction(t('error.walletConnect.cannotDetermineChain', { dAppName: name })));
```

Comment:
Could we please add a sentry / error log here so we can track when this has happened. Thanks